lib/api/utilities/functions.php

```diff
@@ -375,7 +374,7 @@ function beans_count_recursive( $array, $depth = false, $count_parent = true ) {
 		return count( $array, COUNT_RECURSIVE );
 	}
 
-	$count = $count_parent ? count( $array ) : 0;
+	$count = $count_parent ? count($array) : 0;
```

Comment:
This seems like it goes against WPCS?

Reply:
Yup.  Pushed a fix for it.  There's also a question that I'm working with @ThierryA  on to understand the intent of `$count_parent`.  But we can do that in the next PR.